### DISCUSSION TITLE
关于passwall2更新节点后分流消失的问题

### DISCUSSION THREAD

**Discussion by xsmooyi** [Openwrt-Passwall/openwrt-passwall2#234] [Q&A]
请问下大家，在passwall2更新节点后，我的分流选项消失了，重新删除-添加节点后分流选项也没有出现。

不知道是什么原因引起的...

纯小白一枚，希望各位大佬可以帮助解决问题，谢谢TAT

<img width="1114" alt="image" src="https://user-images.githubusercontent.com/129568444/229261475-44c091f0-c44f-4b4c-9906-c42bac620282.png">

<img width="1126" alt="image" src="https://user-images.githubusercontent.com/129568444/229261444-b71b3b4d-7d01-4289-9b9d-dfa2bd2e7112.png">

<img width="680" alt="image" src="https://user-images.githubusercontent.com/129568444/229261596-f556092a-2bb7-4466-897b-e915cbe837df.png">

**Comment by handsomeji** [comment]
最新源码还会吗？

**Reply by xsmooyi** [reply]
你好，

我今天更新了ps2，也还是遇到了一样的问题；

另外可以参考的是，今天同步下载了ps，
刚安装好一开始也是有分流选项的，但是删除 - 添加了新的节点之后，分流选项就不见了....
并且重新删除再安装也没有变好....

**Reply by flylabx** [reply]
我也遇到了

**Comment by GeekSquirrel** [comment]
同样问题

**Comment by wsqers** [comment]
分流节点可能更新时被删了；在passwall中，节点列表>添加>类型：xray/sing-box>传输协议：分流，后面依据个人用途选择后添加即可

**Reply by colourfate** [reply]
正解，需要在节点列表里面添加分流
<img width="1289" height="1034" alt="image" src="https://github.com/user-attachments/assets/9645426b-6c74-49a1-ba70-c76becb8e523" />

**Reply by zsp1977** [reply]
给添加方法啊，只写一个正解没方法不会啊
